ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +45,98 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  containsVismet = (title) => {
+    const vismetMetrics = [
+      'ContentfulSpeedIndex',
+      'FirstVisualChange',
+      'LastVisualChange',
+      'PerceptualSpeedIndex',
+      'SpeedIndex',
+      'videoRecordingStart',
+    ];
+    // eslint-disable-next-line array-callback-return
+    const contains = vismetMetrics.filter((metric) => title.includes(metric));
```

Comment:
Using `find()` instead of `filter()` would be faster & shorter.

Reply:
`The find() method returns the value of the first element in the provided array that satisfies the provided testing function.`

Reply:
Is there a problem here? Wouldn't the alternate solution do the same thing?

Reply:
`find` returns the first element satisfying the criteria, `filter` returns all. but yeah, in this context i gues it would do the same thing.

Reply:
> `find` returns the first element satisfying the criteria, `filter` returns all. but yeah, in this context i gues it would do the same thing.

@alexandru-io so why do we execute redundant (& time consuming) calculus instead of preferring this optimization?
As you said in the daily meeting, the overall functionality takes between 2-3 seconds. Every improvements counts, especially if it's not costing too much implementation time.

Reply:
this code is outdated. please see the new code in "files changed" tab.